### ISSUE TITLE
Ruby errors

### ISSUE THREAD

**Issue by habamax** [asciidoctor/asciidoctor-pdf#996]
Git cloned current master branch.

While trying to create a pdf I get the error:

```
"C:\prg\Ruby26-x64\lib\ruby\2.6.0\tempfile.rb" [unix] 354L, 11484C
(1 of 30): in `<top (required)>': superclass mismatch for class Tempfile (TypeError)
```

![image](https://user-images.githubusercontent.com/234774/51316899-194de600-1a67-11e9-81fa-22f8c07c59d2.png)

PS

About 2 months ago, I did the same and it worked just fine...

With the gem version of the asciidoctor-pdf everything works just fine.

**Comment by habamax** [comment]
Ruby 2.5 has the error too:

![image](https://user-images.githubusercontent.com/234774/51319609-992b7e80-1a6e-11e9-8495-07d514918fff.png)

**Comment by mojavelinux** [comment]
It looks like I had the override logic the wrong way around. Can you test again?

Also, please add the `--trace` option when invoking asciidoctor-pdf.

**Comment by mojavelinux** [comment]
Ruby 2.6 was defining Tempfile in a way that was incompatible with our monkeypatch. It looks like the same was true for Ruby 2.5 too.

**Comment by habamax** [comment]
I have tried without `--trace` and everything works just fine.

Do you need me to run it with `--trace` too?

**Comment by mojavelinux** [comment]
No, that's only for when something goes wrong.

**Comment by mojavelinux** [comment]
resolved by b9edc47238fb7e16c2d76fe329f2e7c0b1b256bc
